lib/wasm/WebAssemblyParser.js

```diff
@@ -62,11 +76,31 @@ class WebAssemblyParser extends Tapable {
 		// parse it
 		const ast = decode(binary, decoderOpts);
 
+		// track offset in module of exported elements
+		const exportsByModuleOffset = {};
+
+		// track offset in module of import elements
+		const importByModuleOffset = {};
+
+		let importOffsetByType = {
+			Global: 0
+		};
+
 		// extract imports and exports
 		const exports = (state.module.buildMeta.providedExports = []);
+		const reExports = (state.module.buildMeta.reExports = []);
+
 		t.traverse(ast, {
 			ModuleExport({ node }) {
 				exports.push(node.name);
+
+				const descriptor = node.descr;
+
+				if (isGlobalExport(node)) {
```

Comment:
Currently it's limited to Globals, we can do the same for functions but it's less problematic at the moment.